edgePy/DGEList.py

```diff
@@ -353,22 +354,40 @@ def create_DGEList(
         )
 
     @classmethod
-    def create_DGEList_handle(
-        cls, data_handle: StringIO, group_file: Path, **kwargs: Mapping
+    def create_DGEList_data_file(
+        cls, data_file: Path, group_file: Path, **kwargs: Mapping
```

Comment:
Does using Path instead of StringIO limit the use of smart_opens use of AWS S3, Hadoop and HTTP server paths?

Reply:
It shouldn't - there's now one function for path and one for StringIO.... either way, both are now present, and we can use the Smart Open version preferentially, if that's desired.  (the create_DGEList_data_file() function is just a wrapper around the create_DGEList_handle() version.)